pkg/odo/cli/service/ui/ui_test.go

```diff
@@ -0,0 +1,85 @@
+package ui
+
+import (
+	"fmt"
+	"github.com/Netflix/go-expect"
+	scv1beta1 "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/v1beta1"
+	"github.com/redhat-developer/odo/pkg/testingutil"
+	"github.com/stretchr/testify/require"
+	"gopkg.in/AlecAivazis/survey.v1/core"
+	"gopkg.in/AlecAivazis/survey.v1/terminal"
+	"k8s.io/apimachinery/pkg/runtime"
+	"testing"
+)
+
+func init() {
+	// disable color output for all prompts to simplify testing
+	core.DisableColor = true
+}
+
+func TestEnterServicePropertiesInteractively(t *testing.T) {
+	t.Skip("TODO: Skip this test until we can figure out what is wrong with it")
```

Comment:
We should add a `// TODO` comment so we don't forget about this here.. Gofmt / golint will automatically pick up on this if we were to revisit this in the future.